mat64/dense.go

```diff
@@ -5,7 +5,10 @@
 package mat64
 
 import (
+	"bytes"
+
 	"github.com/gonum/blas/blas64"
+	"github.com/gonuts/binary"
```

Comment:
But it give use the wrong behaviour for *Dense and we don't want to support SVD etc. The writing that's done for Triangular is just a few lines.